### PR TITLE
Corrected English localization

### PR DESCRIPTION
**Pull request by abledbody**
Updated the English localization to resemble the English version of the Settlers 3 more, as well as correcting grammatical errors.

### CODE REVIEW

File: jsettlers.common/resources/localization/labels_en.properties [hunk 1]
```diff
@@ -147,45 +148,45 @@ controlpanel_weapons_title = Weapons
...
+click_to_move = Click to move this settler.
+abort = cancel
+really_destroy_building = Are you sure you want to destroy this building?
 priority_HIGH = Set priority to high.
 priority_LOW = Set priority to normal.
 priority_STOPPED = Stop working
 building-build-in-progress = Construction site for: %s
 building-material-count = %d
 building-material-required = %d / %d
-stop = stop
```
Comment (michaelzangl) [inline jsettlers.common/resources/localization/labels_en.properties]: Currently, our stop is not the same as hold position. Geologists will just stand around, but soldiers will just stop walking but still attack (and walk away for attacking).

File: jsettlers.common/resources/localization/labels_en.properties [hunk 2]
```diff
@@ -13,128 +13,129 @@
...
+building_LOOKOUT_TOWER = Lookout Tower
+building_TOWER         = Guard Tower
+building_BIG_TOWER     = Large Guard Tower
+building_CASTLE        = Castle
+building_HOSPITAL      = Healer's Quarters
+building_BARRACK       = Barracks
+building_DOCKYARD      = Landing Dock
+
+building_TEMPLE        = Small Temple
+building_BIG_TEMPLE    = Large Temple
```
Comment (michaelzangl) [inline jsettlers.common/resources/localization/labels_en.properties]: We soon also have:
building_MARKET_PLACE

**Comment by michaelzangl** [comment]
Thanks. This also fixes #149 and makes the UI much nicer ;-)

**Comment by andreas-eberle** [comment]
Many thanks for these corrections. @abledbody: Is this ready for merge from your side?

**Comment by abledbody** [comment]
@andreas-eberle I think so, yeah.

**Pull request merged**